src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -248,7 +248,13 @@ private void OnConnectionChanged(object sender, bool connected)
 
                     // s_lastRemoteClientTask info should be saved in the dump
                     // report NFW when connection is closed unless it is proper shutdown
-                    FatalError.ReportWithoutCrash(new Exception("Connection to remote host closed"));
```

Comment:
@sharwell That would depend on the modification. Bucketing depends on a system of heuristics that we largely do not control, though process name, assembly name, exception name, and call stack figure prominently. If we leave those alone bucketing probably stays the same; if we change them, reports will almost certainly start flowing into other buckets.

At the moment changing the bucket isn't really a problem, though, as the places I've fixed up are basically producing no useful data. This change is really just about making all the places we try to report a non-fatal error actually work on the most basic level. :-)

Did I answer your question, or were you referring to something else?